app/views/content/blog/the-head-of-science-inspiring-students-in-blackpool.md

```diff
@@ -0,0 +1,45 @@
+---
+title: The Head of Science inspiring students in Blackpool
+date: "2022-02-09"
+author: Jessica Walmsley
+images:
+  jessica-walmsley:
+    path: "media/images/content/blog/jessica-walmsley.jpg"
+    thumbnail_path: "media/images/content/blog/thumbnails/jessica-walmsley.jpg"
+description: |-
+  Head of Science Jessica Walmsley shares how she brings her passion for science into her Blackpool classroom.
```

Comment:
Could we maybe make this a little longer? Ideally, meta descriptions should be between 120 and 160 characters (gives the user and Google more info about the page)

Reply:
Hi Ellie, I've made it a bit longer - let me know if it needs changing!

Reply:
@emilytraff hey Emily, sorry it's just a little too long now! How about 'Head of Science Jessica Walmsley shares how she brings her passion for science into her Blackpool classroom and why she decided to become a teacher.' ?

Reply:
@EllieNodder Sorry - I've made that change!